authors/apps/articles/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from datetime import datetime, timedelta
+from ..authentication.models import User
+from django.template.defaultfilters import slugify
+from django.utils.text import slugify
+
+
+class Article(models.Model):
+    """
+    Class Implementing The Article Model.
+    """
+
+    title = models.CharField(db_index=True, max_length=255)
+    author = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
+    body = models.TextField(db_index=True)
+    description = models.CharField(db_index=True, max_length=255)
+    slug = models.SlugField(
```

Comment:
I'm thinking the `description` field should be a text field. It should not be limited to 255 characters.

Reply:
Just as you are thinking, I also though it was necessary to have limits to the characters required for each field and that is why I included the 255 character limit. Having made the body to have a 'text' field, making the description 'text' field means we are not considering the database requirements. The description would to some extent be large than the body itself.